packages/data/src/registry.js

```diff
@@ -185,5 +185,9 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 		parent.subscribe( globalListener );
 	}
 
+	if ( parent ) {
+		parent.subscribe( globalListener );
+	}
+
```

Comment:
Why subscribe to the parent twice? Looks like this might have come from a bad merge.

Reply:
Good catch :)